<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dxfplotter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource/resource.qrc">
    <normaloff>:/dxfplotter.png</normaloff>:/dxfplotter.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>469</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusrgd">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
    <addaction name="actionSaveAsFile"/>
    <addaction name="actionExportFile"/>
    <addaction name="actionExportAsFile"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionOpenSettings"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionLeftCutterCompensation"/>
    <addaction name="actionRightCutterCompensation"/>
    <addaction name="actionResetCutterCompensation"/>
    <addaction name="separator"/>
    <addaction name="actionHideSelection"/>
    <addaction name="actionShowHidden"/>
    <addaction name="separator"/>
    <addaction name="actionTransformSelection"/>
    <addaction name="actionMirrorSelection"/>
   </widget>
   <addaction name="menusrgd"/>
   <addaction name="menuEdit"/>
   <addaction name="menuConfiguration"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionSaveFile"/>
   <addaction name="actionExportFile"/>
   <addaction name="actionExportAsFile"/>
   <addaction name="separator"/>
   <addaction name="actionLeftCutterCompensation"/>
   <addaction name="actionRightCutterCompensation"/>
   <addaction name="actionResetCutterCompensation"/>
   <addaction name="separator"/>
   <addaction name="actionTransformSelection"/>
   <addaction name="actionMirrorSelection"/>
   <addaction name="separator"/>
   <addaction name="actionHideSelection"/>
   <addaction name="actionShowHidden"/>
   <addaction name="separator"/>
   <addaction name="actionOpenSettings"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/document-open.svg</normaloff>:/icons/icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/application-exit.svg</normaloff>:/icons/icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExportFile">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/document-export.svg</normaloff>:/icons/icons/document-export.svg</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionOpenSettings">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/configure.svg</normaloff>:/icons/icons/configure.svg</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionLeftCutterCompensation">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/path-outset.svg</normaloff>:/icons/icons/path-outset.svg</iconset>
   </property>
   <property name="text">
    <string>Left Cutter Compensation</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
  <action name="actionRightCutterCompensation">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/path-inset.svg</normaloff>:/icons/icons/path-inset.svg</iconset>
   </property>
   <property name="text">
    <string>Right Cutter Compensation</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionResetCutterCompensation">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/show-path-outline.svg</normaloff>:/icons/icons/show-path-outline.svg</iconset>
   </property>
   <property name="text">
    <string>Reset Cutter Compensation</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionHideSelection">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/layer-visible-off.svg</normaloff>:/icons/icons/layer-visible-off.svg</iconset>
   </property>
   <property name="text">
    <string>Hide Selection</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionShowHidden">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/layer-visible-on.svg</normaloff>:/icons/icons/layer-visible-on.svg</iconset>
   </property>
   <property name="text">
    <string>Show Hidden</string>
   </property>
   <property name="toolTip">
    <string>Show Hidden</string>
   </property>
   <property name="shortcut">
    <string>Shift+H</string>
   </property>
  </action>
  <action name="actionSaveFile">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/document-save.svg</normaloff>:/icons/icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAsFile">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/document-save-as.svg</normaloff>:/icons/icons/document-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionTransformSelection">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/transform-rotate.svg</normaloff>:/icons/icons/transform-rotate.svg</iconset>
   </property>
   <property name="text">
    <string>Transform Selection</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionExportAsFile">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/document-export.svg</normaloff>:/icons/icons/document-export.svg</iconset>
   </property>
   <property name="text">
    <string>Export As</string>
   </property>
   <property name="toolTip">
    <string>Export As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionMirrorSelection">
   <property name="icon">
    <iconset resource="../../resource/resource.qrc">
     <normaloff>:/icons/icons/resizecol.svg</normaloff>:/icons/icons/resizecol.svg</iconset>
   </property>
   <property name="text">
    <string>Mirror Selection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resource/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
